test/garbage_collection_test.js

```diff
@@ -26,7 +26,12 @@ suite('garbage collection tests', function () {
       delayFactor: 15 * 1000,
       randomizationFactor: 0.25
     },
-    log: logger()
+    log: logger(),
+    stats: {
+      timeGen: async function (series, fn) {
+        return await fn;
```

Comment:
We're not recording timings (and don't care about them either) here in the test.  This is mainly just so that when we use timeGen elsewhere in the image manager stuff it doesn't puke saying that timeGen is not a function when it attempts to use it.